src/services/subject.js

```
@@ -0,0 +1,38 @@
+    const [subjects, total] = await Promise.all([
+      Subject.find(query)
+        .skip(skip)
+        .limit(limit)
+        .lean()
+        .populate('category', 'name appearance')
+        .exec(),
+      Subject.countDocuments(query)
+    ]);
+
+    const subjectsWithCategoryNames = subjects.map(subject => ({
+      ...subject, 
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)